src/app/ui/components/containers/dialog/dialog.tsx

```diff
@@ -9,12 +9,12 @@ import { pxStringToNumber } from '@shared/utils/px-string-to-number';
 
 export interface DialogProps {
   isShowing: boolean;
-  onClose(): void;
+  onClose?(): void;
 }
 interface RadixDialogProps extends DialogProps {
   children: ReactNode;
   footer?: ReactNode;
-  header: ReactElement<any, string | JSXElementConstructor<any>>;
+  header?: ReactElement<any, string | JSXElementConstructor<any>>;
```

Comment:
I tried changing the type of this to `ReactNode` but I need these types as I am using `cloneElement` to clone the `header` passed to the `Dialog` and pass on the `onClose` action